model/client4.go

```diff
@@ -1035,10 +1039,23 @@ func (c *Client4) CreateUserAccessToken(userId, description string) (*UserAccess
 	}
 }
 
-// GetUserAccessToken will get a user access token's id, description and the user_id
-// of the user it is for. The actual token will not be returned. Must have the
-// 'read_user_access_token' permission and if getting for another user, must have the
-// 'edit_other_users' permission.
+// GetAllUserAccessToken will get a all access token's id, description, is_active
```

Comment:
Minor grammar and text updates `a all access token's` -> `a page of access tokens'`

Reply:
Fixed and also fixed in `GetUserAccessToken` below.